### ISSUE TITLE
feat(h200): H200 GRPO training pipeline setup

### ISSUE THREAD

**Issue by overthetopseo** [OTT-Cybersecurity-LLC/lyrie-ai#53]
## H200 Training Pipeline

Set up the Lyrie fine-tuning pipeline on the H200 (140GB VRAM, Axolotl pre-installed).

**Server:** ssh h200 (210.157.233.86:38084) | vast.ai ID: 35732872
**Framework:** Axolotl (pre-installed) + GRPO via TRL

**Phase 1 — Training data pipeline:**
```
/workspace/lyrie-llm/
  configs/
    lyrie-cyber-7b.yaml    # Axolotl GRPO config
    lyrie-seo-7b.yaml
  datasets/
    cyber-outcomes.jsonl   # scored conversations
    seo-outcomes.jsonl
    trading-outcomes.jsonl
  models/                  # checkpoints
  runs/                    # training logs
  scripts/
    collect_training_data.sh  # pulls from Mac1 outcomes.jsonl
    train.sh                  # launches Axolotl
    eval.sh                   # runs eval suite
```

**Base model:** Qwen3-7B-Instruct (fits comfortably in 140GB, leaves room for training)

**Domain-specific reward functions:**
```python
def cyber_reward(completion, reference):
    # Was threat correctly classified?
    # Was CVE severity accurate?
    # Did PoC match expected behavior?

def seo_reward(completion, reference):
    # Was content published successfully?
    # Readability score >= 80?
    # Keyword density correct?

def trading_reward(completion, reference):
    # Was signal direction correct?
    # Was risk properly sized?
```

**Training schedule:** Idle-time only (2-4AM Dubai = 10PM-midnight UTC)

**Tasks:**
- [ ] Set up /workspace/lyrie-llm directory structure on H200
- [ ] Write Axolotl config for GRPO fine-tuning
- [ ] Data collection script (pulls outcomes.jsonl from Mac1 nightly)
- [ ] Reward function implementations per domain
- [ ] Training launch script with WandB logging
- [ ] Eval suite against Lyrie benchmark tasks
- [ ] Model push to HuggingFace private repo
- [ ] 4AM cron: collect data → H200 → train → eval → report

**Comment by overthetopseo** [comment]
Shipped in v0.5.0 ✅
